examples/cluster/plot_kmedoids_digits.py

```diff
@@ -0,0 +1,78 @@
+"""
+=============================================================
+A demo of K-Medoids clustering on the handwritten digits data
+=============================================================
+
+In this example we compare different pairwise distance
+metrics for K-Medoids.
```

Comment:
I'd rather update the clustering gallery example:

http://scikit-learn.org/stable/auto_examples/cluster/plot_cluster_comparison.html

unless there is a take home message to convey using digits